tutorials/7-cosmjs/4-with-keplr.md

```diff
@@ -360,7 +355,7 @@ onSendClicked = async(e: MouseEvent<HTMLButtonElement>) => {
     // Suggest the testnet chain to Keplr
     await keplr.experimentalSuggestChain(this.getTestnetChainInfo())
     // Create the signing client
-    const offlineSigner: OfflineSigner =
+    const offlineSigner =
```

Comment:
I prefer to leave this in to give additional information. Compiler errors can be fixed with the proper `import`.

Reply:
That's the thing, `OfflineSigner` is no longer a valid type. Maybe it should then be updated to `OfflineAminoSigner & OfflineDirectSigner` as that's the type returned by `window.getOfflineSigner`

Reply:
Ah, then the hands-on exercise will be updated in time when we review the whole CosmJS exercise. And for this unlinked code, we might as well change like your suggestion.